src/transpiler/babel.ts

```diff
@@ -105,16 +105,6 @@ export function getDefaultBabelOptions({browserslist, transpilerOptions}: GetDef
 				  ])
 		],
 		plugins: [
-			// If preset-env will be included, shipped proposals will be included already. If not, apply them here
```

Comment:
All of the changes you've made here has been done by my self in the meantime, including swapping the comments, so to avoid merge conflicts I think these changes should be removed. If you pull in the latest changes from master, I think it will sort itself nicely.